### DISCUSSION TITLE
State of the licence & eleventy branch

### DISCUSSION THREAD

**Discussion by julientaq** [thegetty/quire#242] [General]
Hi folks!

I work on [Pagedjs](https://pagedjs.org), an open source js lib to make books (and stuff that can be printed) from of HTLM. We got a proposition from a client to setup a single source publishing setup, and we’re trying to see if Quire can be part of a setup.

The problem is that, even if the code is open, the licence don’t really give any clear answers about that.  
What licence is quire based on? 

Also, i found this experiment/eleventy branch (which is very exciting as eleventy is quite amazing), so shout out, we’d be happy to contribute on that bit  :-)

**Comment by geealbers** [comment]
Hi @julientaq, thanks for reaching out. We're fans of pagedjs and have our eyes on it for future Quire development! As to you question, our plan is to release Quire later this year under a gnu public license v3. Until then our current license terms are [here](https://github.com/thegetty/quire/discussions/55). Assuming you are working on behalf of the client and won't be building a new tool with Quire in it that would then be distributed beyond that client, I think you're covered and I'd encourage you to move ahead. If though you or your client need a more explicit license to cover your needs, please reach out to us at quire@getty.edu and we'd be happy to work with you on something.

And yes, we're experimenting with an 11ty version of Quire now and it's looking pretty good. Really nice to have your early vote of confidence and enthusiasm! You may hear more from us on that soon. :)

**Reply by julientaq** [reply]
Ah cool!
The thing is we were ready to rebuild something becaure 11ty allows much more freedom than hugo (and i liked hugo a lot).
But if you’re going the 11ty  road, and your licence is that one i guess it would make more sense that we contribute and work together.
I’ll discuss with the team, and we’ll reach out to you. :)

**Reply by geealbers** [reply]
Sounds good! It would be exciting to combine forces. I would also be happy to arrange a conversation with your team and ours too if that would help work out the options, but we'll wait to hear from you.

**Reply by geealbers** [reply]
@julientaq, one of our engineers just reminded me that we weren't planning on using GPL v3 license, but rather the [BSD 3](https://opensource.org/licenses/BSD-3-Clause). Sorry about that.

**Reply by julientaq** [reply]
Hi there!
I sent you a email, did you get it?
Thanks!

**Reply by geealbers** [reply]
Apologies, it went to spam for some reason but I've got it now. I'll bring it to the team on Monday and we can set up a time for us all to chat. Thanks!
